### PR TITLE
feat(Integrations): Created a new page for the Atlassian integration

### PR DESCRIPTION
**Pull request by nbaenam**
PR to create a new page explaining how to integrate New Relic with Atlassian Jira.

This is the ticket: NR-236395

### CODE REVIEW

**Comment by nbaenam** [comment]
Although it seems that some checks failed, this PR was built correctly: [https://app.netlify.com/sites/docs-website-netlify/deploys/6627913e39670d0598ba13e7](https://app.netlify.com/sites/docs-website-netlify/deploys/6627913e39670d0598ba13e7)

File: src/content/docs/more-integrations/best-practices-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: Best practices for integration users
+tags:
+  - Best practices
+  - Integrations
+metaDescription: Here are some best practices for controlling how you expose your New Relic data when you're using an integration.
+freshnessValidatedDate: 2024-03-25
+---
+
+As a New Relic administrator, you must operate your New Relic organization with the principle of least privilege. By following this principle, you can reduce the risk of unauthorized access to sensitive data. This includes assigning permissions based on job roles and responsibilities. In a [New Relic organization](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/), account-based access allows you to control what information is available to the users in your organization. This ensures that sensitive data remains secure and only authorized personnel can view it.
```
Comment (jeff-colucci) [inline src/content/docs/more-integrations/best-practices-integration.mdx:10]: I think we may need a brief description of what "principle of least privilege" is here

File: src/content/docs/more-integrations/best-practices-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+tags:
+  - Best practices
+  - Integrations
+metaDescription: Here are some best practices for controlling how you expose your New Relic data when you're using an integration.
+freshnessValidatedDate: 2024-03-25
+---
+
+As a New Relic administrator, you must operate your New Relic organization with the principle of least privilege. By following this principle, you can reduce the risk of unauthorized access to sensitive data. This includes assigning permissions based on job roles and responsibilities. In a [New Relic organization](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/), account-based access allows you to control what information is available to the users in your organization. This ensures that sensitive data remains secure and only authorized personnel can view it. 
+
+Dealing with integrations with third-party systems can complicate application access. For example, users in the other system may not have access to New Relic. The system may also lack centralized permissions or controls. Learn about best practices for data sharing on this page. Take control of how you want to expose your New Relic data during integrations.
```
Comment (jeff-colucci) [inline src/content/docs/more-integrations/best-practices-integration.mdx:12]: Some more description on what this "other system" is, the wording is kind of confusing.

I think you can cut the last sentence as well.

File: src/content/docs/more-integrations/best-practices-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Some integrations with New Relic take advantage of the ability to generate an API key to expose your data to other systems (for example, the Atlassian integration). If a user has access to 3 out of 5 accounts in a New Relic organization, their generated API key will have access to those same 3 accounts. 
+
+<Callout variant="important">
+Note that an API key inherits the account permissions of the user who generated it.
+</Callout>
+
+It's important to understand this distinction when setting up an integration. For example, you may want to make sure about generating an API key from a user who has the correct permissions. You may want the user to only have access to certain accounts in the application. This limits what they can see and share with others. For example, say you have more users in your Atlassian application than in your New Relic accounts, and you configure the Atlassian integration. The users in the Atlassian integration will be able to see the New Relic data shared with Atlassian in the Atlassian application. Regardless of whether those users have access to the corresponding New Relic accounts.
+
+
+## Integration user leverage [#leverag-integration-user]
```
Comment (jeff-colucci) [inline src/content/docs/more-integrations/best-practices-integration.mdx:27]: This title is likely accurate but it's a little confusing at first glance, anyway to clarify it?

File: src/content/docs/more-integrations/best-practices-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+When configuring third-party integrations with New Relic, we highly recommend using a dedicated integration user. This means creating a New Relic user just for a specific integration. You can give this user a name that represents the integration in question.
+
+When configuring third-party integrations with New Relic, we highly recommend using a dedicated integration user. This means [creating a New Relic user](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions/) just for a specific integration. You can give this user a name that represents the integration in question. A big advantage of this is that you do not need to use your Admin-level access for the integration. All you have to do is create the new user and assign only the accounts that you want to expose to the integration. Then, generate an API key for the integration user. This allows the API key that you use for an integration to expose only the account details that you want to use for the integration. It also allows you to decouple your personal New Relic access from that of the integration.
```
Comment (jeff-colucci) [inline src/content/docs/more-integrations/best-practices-integration.mdx:31]: do not > don't

File: src/content/docs/more-integrations/best-practices-integration.mdx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+Some integrations with New Relic take advantage of the ability to generate an API key to expose your data to other systems (for example, the Atlassian integration). If a user has access to 3 out of 5 accounts in a New Relic organization, their generated API key will have access to those same 3 accounts. 
+
+<Callout variant="important">
+Note that an API key inherits the account permissions of the user who generated it.
+</Callout>
+
+It's important to understand this distinction when setting up an integration. For example, you may want to make sure about generating an API key from a user who has the correct permissions. You may want the user to only have access to certain accounts in the application. This limits what they can see and share with others. For example, say you have more users in your Atlassian application than in your New Relic accounts, and you configure the Atlassian integration. The users in the Atlassian integration will be able to see the New Relic data shared with Atlassian in the Atlassian application. Regardless of whether those users have access to the corresponding New Relic accounts.
```
Comment (jeff-colucci) [inline src/content/docs/more-integrations/best-practices-integration.mdx:24]: The last sentence should be joined with the one before it: "in the Atlassian application, regardless of whether"

File: src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+import opentelemetryAtlassianNRapiToken from 'images/opentelemetry_screenshot-crop_connect-nr-jira.webp'
+
+import opentelemetryJiraIncidentManagement from 'images/opentelemetry_screenshot-full_jira-incident-mgt.webp'
+
+import opentelemetryJiraNRConfiguration from 'images/opentelemetry_screenshot-full_jira-nr-configuration.webp'
+
+
+
+New Relic offers an integration with Atlassian Jira that enables the incident tab feature of Jira. This feature syncs all your New Relic issue and incident information with Atlassian. This data will then appear in Jira on the incidents tab and will let you connect it with Jira issues to improve your outage workflows. The sections outline how you set up this integration and the best practices for usage.
```
Comment (jeff-colucci) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx:21]: I'd say "feature within Jira", I think

Comment (jeff-colucci) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx:21]: This data will then appear > This data then appears

File: src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+## Data sharing [#data-sharing]
+
+Once you install the integration, you allow New Relic for Jira to perform these actions:
+
+* Read entities, incidents, and issues from Jira.
+* Write entities, incidents, and issues to Jira.
+* Delete entities, incidents, and issues from both New Relic and Jira to mirror data reporting with each entity.
+
+New Relic data may be visible to non-New Relic users within your organization. These users can view information through the Jira application.
```
Comment (jeff-colucci) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx:36]: Is this necessary with the callout below? It feels like they say the same thing essentially

File: src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+
+<Callout variant="caution">
+  This integration will expose your New Relic issues and incident information to all users in the related Jira workspace. Jira users in this workspace may not have access to New Relic, and even if they do, they may not have access to the same New Relic accounts as your API key user. 
+  
+  If you have New Relic account information to hide, use an integration user that doesn't have access to your sensitive accounts.
```
Comment (jeff-colucci) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx:41]: Should we link out from "integration user" to the part of the docs that covers what that is?

File: src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+Create and manage your API keys from the [New Relic API keys UI](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher) page or log in to the [Atlassian API tokens](https://id.atlassian.com/manage-profile/security/api-tokens) page and click **Create API token**.
+
+</Step>
+
+<Step>
+## Apply a New Relic API key [#apply-new-relic-key]
+
+Once you've created an API key, you can apply that key to the first page of the New Relic connect application. 
+
+Paste your New Relic API token and click **Connect**.
```
Comment (jeff-colucci) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx:75]: I would be very specific about where this token is pasted in to, if you can

File: src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+After you've selected your New Relic entities, the application starts syncing data from New Relic into Atlassian Jira. Open the incidents tab to see a list of detected incidents.
+
+
+</Step>
+
+</Steps>
+
+## Enable the incidents tab [#enable-incidents-tab]
+
+You may need to link to the Atlassian documentation. It walks a Jira site admin through enabling the Incidents tab and assigning it to a workspace. There is a chance they are going to enable by default and then, you won't need this section.
```
Comment (jeff-colucci) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx:111]: I notice that Incidents and incident tab and the like have some inconsistent capitalization that will need to be addressed

Comment (jeff-colucci) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx:111]: There is > There's

Comment (jeff-colucci) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx:111]: "a chance they are going to enable by default" reads very awkwardly to me

File: src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+
+
+## Remove a connection between New Relic and Atlassian Jira [#remove-connection]
+
+You can stop syncing data from New Relic at any time. and removing the synced data. You can do this from your Atlassian Marketplace configuration page being a Jira site administrator. You can select your connected organization and click **Delete**. After it finishes, you will no longer see New Relic incidents and issue information present in the incidents tab in Jira.
```
Comment (jeff-colucci) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx:116]: These first two sentences look like they should be one sentence?

File: src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+
+## Connect to a different organization or user [#different-conexion]
+
+At any point, you may connect to the wrong New Relic organization. Or, you may want to adjust the account access your API key provides. You can do this from your Atlassian Marketplace page. When you click the **Connect another New Relic organization**, you'll need to provide a New Relic API key. Giving a new key will delete your current connection. This includes all data synced to Atlassian. Then, a new sync process will start with the account and organization access of the new API key.
```
Comment (jeff-colucci) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx:120]: Again, I'd merge the first two sentences

**Review by jeff-colucci — DISMISSED** [review]
I've made some comments but once those are addressed I think you're good to publish

File: src/content/docs/more-integrations/best-practices-integration.mdx (hunk 5 above)
Comment (nbaenam) [reply src/content/docs/more-integrations/best-practices-integration.mdx:24]: Yes, much better. Thanks!

File: src/content/docs/more-integrations/best-practices-integration.mdx [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+
+
+## Integration user leverage [#leverag-integration-user]
+
+When configuring third-party integrations with New Relic, we highly recommend using a dedicated integration user. This means creating a New Relic user just for a specific integration. You can give this user a name that represents the integration in question.
```
[the remark this answers is not in the document]
Comment (nbaenam) [reply src/content/docs/more-integrations/best-practices-integration.mdx:29]: ouch...you're right. Deleting the first one.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx (hunk 7 above)
Comment (nbaenam) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx:36]: No, I don't think it's necessary. I'm going to remove it. Thanks!

**Comment by nbaenam** [comment]
I'm going to merge because this has been already reviewed by Jeff and 2 SMEs.

**Pull request merged**